cmd/soroban-cli/src/output.rs

```diff
@@ -0,0 +1,31 @@
+pub struct Output {
+    pub quiet: bool,
+}
+
+impl Output {
+    pub fn new(quiet: bool) -> Output {
+        Output { quiet }
+    }
+
+    fn print(&self, icon: &str, message: &str) {
+        if !self.quiet {
+            eprintln!("{icon} {message}");
+        }
+    }
+
+    pub fn check(&self, message: &str) {
+        self.print("✅", message);
+    }
+
+    pub fn info(&self, message: &str) {
+        self.print("ℹ️", message);
+    }
+
+    pub fn globe(&self, message: &str) {
+        self.print("🌎", message);
+    }
+
+    pub fn link(&self, message: &str) {
+        self.print("🔗", message);
+    }
```

Comment:
```suggestion
    fn print<T: Display>(&self, icon: &str, message: T) {
        if !self.quiet {
            eprintln!("{icon} {message}");
        }
    }

    pub fn check<T: Display>(&self, message: T) {
        self.print("✅", message);
    }

    pub fn info<T: Display>(&self, message: T) {
        self.print("ℹ️", message);
    }

    pub fn globe<T: Display>(&self, message: T) {
        self.print("🌎", message);
    }

    pub fn link<T: Display>(&self, message: T) {
        self.print("🔗", message);
    }
```

This will allow passing `String`, or `&str` as well as any type that implements `Display`.